src/components/Category/GroupedCategoryOption.tsx

```diff
@@ -0,0 +1,108 @@
+import { useCallback, useEffect, useMemo, useState } from 'react'
+
+import classNames from 'classnames'
+import isNumber from 'lodash/isNumber'
+import toSnakeCase from 'lodash/snakeCase'
+
+import { BiddingProposalType, GovernanceProposalType } from '../../entities/Proposal/types'
+import { getUrlFilters } from '../../helpers'
+import useFormatMessage from '../../hooks/useFormatMessage'
+import useURLSearchParams from '../../hooks/useURLSearchParams'
+import Link from '../Common/Typography/Link'
+import Text from '../Common/Typography/Text'
+import Arrow from '../Icon/Arrow'
+import Governance from '../Icon/ProposalCategories/Governance'
+import Tender from '../Icon/ProposalCategories/Tender'
+import SubItem from '../Icon/SubItem'
+
+import { CategoryOptionProps } from './CategoryOption'
+
+type ValueOf<T> = T[keyof T]
+
+type Props = CategoryOptionProps & {
+  group: ValueOf<typeof GovernanceProposalType>[] | ValueOf<typeof BiddingProposalType>[]
+}
+
+const icons: Record<string, React.ReactNode> = {
+  implementation: <Governance size={24} />,
+  bidding_and_tendering: <Tender size={24} />,
+}
+
+function GroupedCategoryOption({ active, type, className, count, group, ...props }: Props) {
+  const t = useFormatMessage()
+  const params = useURLSearchParams()
+  const currentType = useMemo(() => params.get('type'), [params])
+  const isGroupSelected = useMemo(() => group.includes(currentType as never), [group, currentType])
+
+  const [isGroupExpanded, setIsGroupExpanded] = useState(isGroupSelected)
+
+  useEffect(() => {
+    setIsGroupExpanded(isGroupSelected)
+  }, [isGroupSelected, currentType])
```

Comment:
is `currentType` necessary here? Current Type will change on click / navigation.